### PR TITLE
More beautiful automation!

### PR DESCRIPTION
**Pull request by AmritPatel**

### CODE REVIEW

File: src/hbf/scrapeZillowLinks.py [hunk 1]
```diff
@@ -125,7 +126,7 @@ def scrapeZillowLinks(SoldHomeZillowLinks, headerInput):
                 SoldHomeHeating,
                 SoldHomeCooling,
                 SoldHomeParking,
-                SoldHomeLotSize
+                SoldHomeLotSize,
```
Comment (MattThomas9) [inline src/hbf/scrapeZillowLinks.py:129]: why a comma after the last variable in this list? is this not going to throw an error???

Comment (MattThomas9) [reply src/hbf/scrapeZillowLinks.py:129]: @AmritPatel see comment above

**Review by MattThomas9 — DISMISSED** [review]
see comment about the comma at the end of the list... is it correct? also, trying to see what happens when i do the "request changes" review as opposed to the "approve" review.

File: src/hbf/scrapeZillowLinks.py (hunk 1 above)
Comment (AmritPatel) [reply src/hbf/scrapeZillowLinks.py:129]: It is part of the [style rules](https://black.readthedocs.io/en/stable/the_black_code_style.html#the-magic-trailing-comma) that are PEP8 compliant.

**Review by MattThomas9 — APPROVED** [review]

**Pull request merged**